packages/menu-bar/src/vaadin-menu-bar-mixin.js

```diff
@@ -333,6 +333,10 @@ export const MenuBarMixin = (superClass) =>
         button.textContent = item.text;
       }
 
+      if (item.className) {
+        button.setAttribute('class', item.className);
```

Comment:
nit: no need to do this via attributes. Example https://github.com/vaadin/web-components/blob/af30824761ef47926fb8e9886aba4dcfb1c8f212/packages/avatar-group/src/vaadin-avatar-group.js#L375

Reply:
Done.